code/modules/client/verbs/ooc.dm

```diff
@@ -36,7 +36,7 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 	msg = emoji_parse(msg)
 
-	if((msg[1] in list(".",";",":","#")) || findtext_char(msg, "say", 1, 5))
+	if(SSticker.HasRoundStarted() && (msg[1] in list(".",";",":","#")) || findtext_char(msg, "say", 1, 5))
```

Comment:
http://www.byond.com/docs/ref/#/operator

you messed up the existing check due to order of operations.

Reply:
oops i didn't even notice

Reply:
i looked at that for an entire minute to notice what was wrong lol